apps/frontend/src/app/PageTeam/CharacterDisplay/Tabs/TabOptimize/index.tsx

```diff
@@ -609,242 +538,155 @@ export default function TabBuild() {
-                : t('generateButton.generateBuilds')}
-            </Button>
-          </span>
-        </BootstrapTooltip>
-      </ButtonGroup>
+              <MenuItem>
+                <Typography variant="caption" color="info.main">
+                  {t('threadDropdownDesc')}
+                </Typography>
+              </MenuItem>
+              <Divider />
+              {range(1, defThreads)
+                .reverse()
+                .map((v) => (
+                  <MenuItem key={v} onClick={() => setMaxWorkers(v)}>
+                    <Trans t={t} i18nKey="thread" count={v}>
+                      {{ count: v }} Threads
+                    </Trans>
+                  </MenuItem>
+                ))}
+            </DropdownButton>
+            <BootstrapTooltip placement="top" title={t`notifyTooltip`}>
+              <Button
+                sx={{ borderRadius: 0 }}
+                color="warning"
+                onClick={() => setnotification((n) => !n)}
+              >
+                {notification ? (
+                  <NotificationsActiveIcon />
+                ) : (
+                  <NotificationsOffIcon />
+                )}
+              </Button>
+            </BootstrapTooltip>
+          </ButtonGroup>
+          <BootstrapTooltip
+            placement="top"
+            title={
+              !optimizationTargetNode || optimizationTargetNode.isEmpty
+                ? t('selectTargetFirst')
+                : ''
+            }
+          >
+            <Box display="flex" flexGrow={1} maxHeight="5em">
+              <Button
+                disabled={
+                  !characterKey ||
+                  !optimizationTarget ||
+                  !optimizationTargetNode ||
+                  optimizationTargetNode.isEmpty
+                }
+                color={generatingBuilds ? 'error' : 'success'}
+                onClick={
+                  generatingBuilds
+                    ? () => cancelToken.current()
+                    : generateBuilds
+                }
+                startIcon={generatingBuilds ? <Close /> : <TrendingUp />}
+                sx={{ borderRadius: '0px 0px 4px 4px' }}
+                fullWidth
+              >
+                {generatingBuilds
+                  ? t('generateButton.cancel')
+                  : t('generateButton.generateBuilds')}
+              </Button>
+            </Box>
+          </BootstrapTooltip>
+        </Grid>
+      </Grid>
+      {/* Footer */}
       {!!characterKey && (
         <BuildAlert
           {...{ status: buildStatus, characterName, maxBuildsToShow }}
         />
       )}
-      <Box>
-        <ChartCard
-          disabled={generatingBuilds || !optimizationTarget}
-          plotBase={plotBase}
-          setPlotBase={setPlotBase}
-          showTooltip={!optimizationTarget}
-        />
-      </Box>
-      <CardLight>
+      <Accordion sx={(theme) => ({ bgcolor: theme.palette.contentLight.main })}>
```

Comment:
Since this is the only Accordion component, I'd say we should apply `disableGutters` to remove the additional margin created top&bottom

Reply:
Also, can we apply some corner rounding?